### DISCUSSION TITLE
Is it possible to run Anrdoid apps here?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [Alex313031/ThoriumOS#85] [General]
I read that it is possible on ChromeOS, but not in ChromiumOS.

**Comment by Interested-Deving-1896** [comment]
That is currently incorrect . . .

ChromeBrew can be used to Compile/Build WayDroid.

As well as CROS_SDK & other ways . . .

You can see an overview of other ways located at this Git-Issue . . .

pieroproietti/penguins-eggs#475
